### ISSUE TITLE
Allow tracking heart pieces and heart containers

### ISSUE THREAD

**Issue by Vahdokah** [wooferzfg/tww-rando-tracker#978]
In my time doing dozens of wind waker randomizers, this tracker could use some small changes to make it a lot more user friendly with any setting.
1) Having the option for the shield to be completely greyed out upon tracker generation for those who throw the hero shield into the randomization.
2) Having an Icon for Heart pieces, heart containers, rupees, and spoils pick-ups for those who want to keep track of them.
3) Blue chu jelly locations on the tracker on the opposite side of where the treasure charts are.
I'm sure i have more, but those are the main 3 I can think of

**Comment by wooferzfg** [comment]
1. already works if you paste in the permalink

**Comment by Vahdokah** [comment]
i tried pasting in the permalink, all i get is the blank wave background and nothing else. I'd rather there just be an option on the site to toggle on and off if you put the shield in

**Comment by wooferzfg** [comment]
@Vahdokah What version of the randomizer are you using? 1.9.0?

**Comment by Vahdokah** [comment]
unless it's been updated, then i'm using 1.9.0.

**Comment by wooferzfg** [comment]
Do you get some kind of error in your browser console when the tracker doesn't load? If so, could you show me the error?

**Comment by Vahdokah** [comment]
There's no error, just a backround and that's it
![image](https://user-images.githubusercontent.com/30491453/209764430-d04b52df-72a6-49ae-832a-7076a960ac5f.png)

**Comment by wooferzfg** [comment]
Can you check your browser console? (Right click -> Developer Tools)

Also, it would be helpful if you could send the tracker URL.

**Comment by Vahdokah** [comment]
![image](https://user-images.githubusercontent.com/30491453/209776470-a7c5aac2-b8b4-4b66-9e53-78f3ddfe4a76.png)
https://www.wooferzfg.me/tww-rando-tracker/#/tracker/new/MS45LjAAU2VlZAD///+ADkBAAwAAAAAAAQAA here's the link, and im using opera GX for my browser

**Comment by wooferzfg** [comment]
@Vahdokah Just to be clear, you pasted the permalink into this field on the launcher page (https://www.wooferzfg.me/tww-rando-tracker)?
![image](https://user-images.githubusercontent.com/8262173/209823700-27aedd80-fdbd-4d81-ad38-8c853cd18d7f.png)

The correct URL for the permalink should be https://www.wooferzfg.me/tww-rando-tracker/#/tracker/new/MS45LjAAU2VlZAD%2F%2F%2F%2BADkBAAwAAAAAAAQAA (the `/` characters in the permalink get escaped). Not sure if there's some issue with that behavior on Opera though.

**Comment by Vahdokah** [comment]
yes, and i get that same randomizer screen with the hero shield ticked off like i obtained it. So when i do randomizers with the trackers, i just kinda have to deal with it, which is kinda why i was hoping for an option on the site so i could have it greyed out. Up to you if you want to deal with adding it, but i mainly just put forward my personal suggestions/wants.

**Comment by wooferzfg** [comment]
The permalink you sent me has "Progressive Shield" in the starting items, so if you tried to open the URL I sent, that's why it starts with a shield. Try pasting a permalink that starts without the shield.

**Comment by wooferzfg** [comment]
3. is already covered by this issue: #55

**Comment by wooferzfg** [comment]
Leaving this open for the heart container/heart piece tracking.
